src/main/java/edu/greenblitz/robotName/subsystems/Lifter/LifterInputs.java

```diff
@@ -0,0 +1,14 @@
+package edu.greenblitz.robotName.subsystems.Lifter;
+
+import org.littletonrobotics.junction.AutoLog;
+
+@AutoLog
+public class LifterInputs {
+    public double appliedOutput;
+    public double outputCurrent;
+    public double position;
+    public double velocity;
+    public boolean isSwitchPressed;
+    public boolean isMotorAtPosition;
+    public double destination;
```

Comment:
Is there a reason to use public-access fields and not standard getter/setters?

Reply:
it's easier to use without getters and setters